src/game/option_graphics.c

```diff
@@ -114,10 +117,74 @@ static void Option_GraphicsInitText()
     }
 }
 
+static void Option_GraphicsChangeTextOption(int32_t option_num)
+{
+    char buf[OPTION_LENGTH];
+    char tmp[10];
+
+    switch (option_num) {
+    case TEXT_PERSPECTIVE:
+        sprintf(
+            buf, g_GameFlow.strings[GS_DETAIL_PERSPECTIVE_FMT],
+            g_GameFlow.strings
+                [g_Config.rendering.enable_perspective_filter ? GS_MISC_ON
+                                                              : GS_MISC_OFF]);
+        Text_ChangeText(m_Text[TEXT_PERSPECTIVE], buf);
+        break;
+
+    case TEXT_BILINEAR:
+        sprintf(
+            buf, g_GameFlow.strings[GS_DETAIL_BILINEAR_FMT],
+            g_GameFlow.strings
+                [g_Config.rendering.enable_bilinear_filter ? GS_MISC_ON
+                                                           : GS_MISC_OFF]);
```

Comment:
Man this formatting. I hate clang-format sometimes but it's 20× times better than nothing